Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -30,30 +30,55 @@
       "type": "pickList",
```

Comment:
"The project from which to download the build artifacts from" for helpMarkDown

Reply:
fixed.

Reply:
fixed.